app/assets/stylesheets/oregon_digital/_work-show.scss

```diff
@@ -333,4 +334,24 @@ dl.work-show {
   width: 80%;
   padding: 20px;
   text-align: center;
+}
+
+.bg-collection {
+  background-color: rgb(0,0,0); /* Fallback color */
+  background-color: rgba(0,0,0, 0.4); /* Black w/opacity/see-through */
+}
+
+// Display for button and text in the content mask
+.button-text-mask {
+  color: white;
+  font-weight: bold;
+  position: absolute;
+
+  top: 28%;
```

Comment:
Can this CSS and the HTML overlay be made more consistent with #3115?

- Use `.bg-text` and `.bg-collection` on the surrounding `<div>` & remove `.button-text-mask`
```
        <% if @collection.mask_content.include?('Item Show Page') %>
          <div class='bg-text bg-collection'> ...
```

- Make this transformation bit the styling for `.bg-collection` to override whatever you need to from `.bg-text`.
```
.bg-collection {
  top: 28%;
  left: 50%;
  transform: translate(-50%, -50%);
  z-index: 2;
  width: 80%;
  padding: 10px;
  text-align: center;
}
```

- And continue to use `.blurred-image` without `.bg-collection` on the blur overlay?
```
<div class="<%= 'blurred-image' if @collection.mask_content.include?('Item Show Page') %>" id="thumbnails" data-behavior="masonry-gallery">

...

<button type="button" class="btn btn-primary" onclick="this.parentNode.style.display = 'none'; $('.blurred-image').removeClass('blurred-image')">I Understand</button>
```

Reply:
I'm not sure on this, haven't tested anything. but it would be more consistent with what @straleyb wrote

Reply:
Agreed. Lets normalize it to my PR and then if any edits need to be made in the QA process then it should be straight forward on both pages.